stacks/apps/feeder.yml

```diff
@@ -251,47 +251,49 @@ Resources:
       Policies:
         - PolicyDocument:
             Statement:
-              - Action: sqs:*
+              - Action:
+                  - sqs:ChangeMessageVisibility
+                  - sqs:Get*
+                  - sqs:List*
+                  - sqs:ReceiveMessage
+                  - sqs:SendMessage
```

Comment:
Did this list come from Shoryuken?

Reply:
https://github.com/ruby-shoryuken/shoryuken/wiki/Amazon-SQS-IAM-Policy-for-Shoryuken

It's more permissive than what Shoryuken needs. I wasn't sure if there was anything else touching the queues, given the old `sqs*` policy.